data/the emerald sword.txt

```diff
@@ -0,0 +1,796 @@
+	
+	on offer
+		conversation
+			`You meet up with the Hai archaeologists that will accompany you to human space, an all female team. They introduce themselves to you as Eyama, Rakei, and Desyo.`
+			`	Eyama is a linguist, specializing in the Sheragi language. She playfully greets you in their rather guttural tongue, with strange tones.`
+			`	Rakei is an archaeological engineer, who immediately takes a liking to the cube as soon as she sees it, examining it's structure.`
+			`	Finally, Desyo is a xenoprogramming expert, who studies alien software, and seemingly has a lot of experience with Sheragi code. Apparently, the displays of the Sheragi room of the museum all run in this exotic language, unseen by the usual public.`
+			`	After the brief introduction, they each take their equipment and load them on the cargo bay of your ship. Rakei approaches you after everything is in place.`
+			`	"We want to meet the human archaeologist who uncovered the ruins in the rim. Can you take us there? We will start examining the cube along the way. I think by the time we get there, we'll have something to work with."`
+				accept
+	
+	on complete
+		conversation
+			`It was a long trip, almost halfway across the galaxy. The Hai archaeologists have been friendly to you, giving you updates every time they make a new breakthrough. During this time, the cube, previously thought to be impenetrable, had been disassembled and cleaned. Components that decayed into a mess of oxidized wires and goopy paste over the eons were been identified and removed to substitute for equivalent ones, and a blueprint of the box and its circuits has started to take shape.`
+			`	As soon as you land, you call Albert Foster, who doesn't believe you when you say you got alien archaeologists to help. You then look for the three Hai to figure out how they're going to go unnoticed on a human world. You're surprised to find Eyama wearing long, dark but colorful baggy human clothing with an equally oversized hoodie and a bandanna. You recognize this outfit, very popular with the youth these days, who are generally regarded as delinquents by the previous generation. Additionally, she's carrying two more sets, for her Hai partners as well. It hides her body really well at the cost of making her look like a young boy, and the Hai's relatively short stature doesn't help.`
+			`	"Call me Miss Deus ex Machina, because I thought ahead and bought these from a merchant back home. Perfect disguise. The humans will never notice. How do I look?"`
+			`	You can't disagree that it makes a good disguise, but refrain from telling them exactly how silly they would look in that clothing. Meanwhile, the other two go back to their rooms to get dressed.`
```

Comment:
Seems like there's a lot of text here; should probably add a choice after Eyama asks how she looks.